.eslintrc.js

```diff
@@ -12,6 +12,9 @@ const config = {
       }
     ],
     'import/no-import-module-exports': 'off'
+  },
+  settings: {
+    'import/ignore': ['node_modules', 'src'] // Using CommonJS in src
```

Comment:
Why is this necessary?

Reply:
Because without it, eslint reports
```
/home/glen/code/prettier-eslint/src/__tests__/index.js
  8:9   error  format not found in '../'   import/named
  8:17  error  analyze not found in '../'  import/named
```
even though `../index.js` is perfectly well exporting format and analyze as the unit tests show. The difficulty is that the src directory of prettier-eslint is in CommonJS module format, and the `import/named` rule really only functions with ES modules. An alternative might be to turn off that one particular eslint rule, but since (short of switching pretter-eslint to ES modules, which would also be fine but beyond the scope of this PR) all import checking is doomed to fail, it seemed better to just let eslint know to ignore issues about imports from src altogether.